src/Clock.scss

```
@@ -0,0 +1,3 @@
+.Clock {
```

```suggestion
.clock {
```
usualy class names are lowercased